src/toil/common.py

```diff
@@ -233,9 +232,7 @@ def parseIntList(s):
         setOption("scaleInterval", float)
         setOption("metrics")
         setOption("preemptableCompensation", float)
-        require(0.0 <= self.preemptableCompensation <= 1.0,
-                '--preemptableCompensation (%f) must be >= 0.0 and <= 1.0',
-                self.preemptableCompensation)
+        assert 0.0 <= self.preemptableCompensation <= 1.0, '--preemptableCompensation (%f) must be >= 0.0 and <= 1.0' % self.preemptableCompensation
```

Comment:
This looks like an user error and hence should raise an exception, not be an assert.